app/src/main/resources/org/jboss/hal/client/configuration/subsystem/deploymentscanner/DeploymentScannerView.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/hal/hal.next/develop/spi/src/main/resources/org/jboss/hal/spi/MbuiView.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<!--
+  ~ Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<view>
+    <vertical-navigation id="deploymentscanner-vertical-navigation">
```

Comment:
If there's only one item in the vertical-navigation, I think we can get rid of it. The xml would the look like 

```xml
<item id="deploymentscanner-item" title="Scanners" icon="pficon pficon-repository">
    <metadata address="/{selected.profile}/subsystem=deployment-scanner/scanner=*">
        ....
    </metadata>
</view>
```